app/src/main.js

```diff
@@ -47,8 +55,14 @@ app.on('activate', () => {
   // sometimes, the event is triggered before app.on('ready', ...)
   // then creating new windows will fail
   if (win.browser === null && appIsReady) {
-    win.create({
-      electron, path, electronLocalshortcut, storage, checkForUpdates,
+    fp(3000, (err, port) => {
+      if (err) {
+        throw new Error(err);
+      }
+
+      win.create({
+        electron, path, electronLocalshortcut, storage, checkForUpdates, port,
+      });
```

Comment:
This whole `fp(3000, (err, port) => {` is duplicated twice in this file. Maybe some helper function? Or at least define the default port in one place (a constant).

Reply:
done